src/assets/src/hooks/useOneTrust.ts

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from 'react';
+
+declare global {
+    interface Window {
+      OnetrustActiveGroups?: string;
+      OptanonWrapper?: () => void;
+    }
+  }
+
+export const useOneTrust = (): [boolean, string, (callback: () => void) => void] => {
+    const src = 'https://cdn.cookielaw.org/consent/03e0096b-3569-4b70-8a31-918e55aa20da/otSDKStub.js'
```

Comment:
Ideally we could pass these 2 values in via config rather than hardcode them, maybe with `spa_globals`? I'm not sure how difficulty that would be with this app though. I'm also not sure if these would ever change, but would require a code change if they do.

Reply:
The value "03e0096b-3569-4b70-8a31-918e55aa20da" is defined in line 12.

Reply:
If the `src` and `dataDomainScript`  variables  values is made available from `spa_globals`  (which i am assuming is Django FE variable shared during the launch of the tool) then the variable values `src` and `dataDomainScript` could be passed as React Props to the component.

As Zhen, shared the alpha numeric values seems to be same at this point `src` and `dataDomainScript` and will be in future then we could reuse with some logic.